src/components/Datagrid/cells.ts

```diff
@@ -17,19 +20,30 @@ export function textCell(value: string): GridCell {
   };
 }
 
-export function numberCell(value: number | null): NumberCell {
+export function booleanCell(value: boolean): GridCell {
```

Comment:
Why can other cells take nullish values whereas boolean does not?

Reply:
You can't clear this field, it's just true or false